pkg/rpc/server/server.go

```diff
@@ -712,7 +712,7 @@ func (s *Server) getStorage(ps request.Params) (interface{}, *response.Error) {
 
 	item := s.chain.GetStorageItem(id, key)
 	if item == nil {
-		return nil, nil
```

Comment:
Seems unrelated to this commit.

Reply:
In fact it is, previous scheme allowed the associated test to pass.